fill/sensors/sensor.h

```diff
@@ -13,15 +13,20 @@
 
 class Sensor {
   public:
-    // initializes the ADC
+    // initializes the ADCs
     Sensor();
     virtual ~Sensor() {}
 
-    float Read(uint8_t channel, bool isADC1);
+    float ReadPT1(); 
+    float ReadPT2(); 
+    float ReadLoadCell();
+
   protected:
     // cache for rolling averages
     std::vector<float> cache_;
 
+    float Read(uint8_t channel, bool isADC1);
```

Comment:
It would be more extensible to implement this as a vector of ADCs and have an argument for index into this rather than a bool